Lib/test/test_sysconfig.py

```diff
@@ -387,10 +387,13 @@ def test_triplet_in_ext_suffix(self):
         if re.match('(i[3-6]86|x86_64)$', machine):
             if ctypes.sizeof(ctypes.c_char_p()) == 4:
                 self.assertTrue(suffix.endswith('i386-linux-gnu.so') or
-                                suffix.endswith('x86_64-linux-gnux32.so'),
-                                suffix)
+                                suffix.endswith('x86_64-linux-gnux32.so') or
+                                suffix.endswith('i386-linux-musl.so'),
+                                f'unexpected suffix {suffix!r}')
             else: # 8 byte pointer size
-                self.assertTrue(suffix.endswith('x86_64-linux-gnu.so'), suffix)
+                expected_suffixes = 'x86_64-linux-gnu.so', 'x86_64-linux-musl.so'
+                self.assertTrue(suffix.endswith(expected_suffixes),
+                                f'unexpected suffix {suffix!r}'))
```

Comment:
```suggestion
            self.assertTrue(suffix.endswith(expected_suffixes),
                            f'unexpected suffix {suffix!r}'))
```

Reply:
This seems wrong? `expected_suffxies` is only defined in this branch, moving it out of `else` would trigger `AttributeError` for the 4-byte case.

Reply:
I think there is an `SyntaxError: unmatched ')'` error.

Reply:
There is!  But the suggestions from Brett, taken together, amount to only defining expected_suffixes in the conditional block, and call assertTrue only once outside of the blocks.